src/analysis/normed_space/basic.lean

```diff
@@ -918,6 +921,28 @@ real.norm_of_nonneg (norm_nonneg _)
 @[simp] lemma nnnorm_norm [normed_group α] (a : α) : nnnorm ∥a∥ = nnnorm a :=
 by simp only [nnnorm, norm_norm]
 
+-- `metric.tendsto_at_top` says `∃ , ∀ n ≥ N, ...` while here we use `∃ N, ∀ n > N, ...`,
+-- which is equivalent, but often more convenient.
+lemma normed_group.tendsto_at_top {β : Type*} [normed_group β] {f : ℕ → β} {b : β} :
+  tendsto f at_top (𝓝 b) ↔ ∀ ε, 0 < ε → ∃ N, ∀ n, N < n → ∥f n - b∥ < ε :=
+begin
+  rw tendsto_iff_norm_tendsto_zero,
+  rw metric.tendsto_at_top,
+  fsplit,
+  { intros h ε εpos,
+    obtain ⟨N, h⟩ := h ε εpos,
+    use N,
+    intros n nh,
+    specialize h n (le_of_lt nh),
+    simpa using h, },
+  { intros h ε εpos,
+    obtain ⟨N, h⟩ := h ε εpos,
+    use N+1,
+    intros n nh,
+    specialize h n nh,
+    simpa using h, },
```

Comment:
```suggestion
  rw at_top_basis_Ioi.tendsto_iff metric.nhds_basis_ball,
  { simp [dist_eq_norm] },
  { apply_instance },
  { apply_instance }
```
It'd be user-friendly to have more such lemmas in `normed_space.basic` (there are already a few, such as `normed_group.tendsto_nhds_zero` and `normed_group.tendsto_nhds_nhds`).  In this case, though, it might work out nicely to _not_ use this lemma, specifically to never rewrite the `at_top` filter in your main Bernstein lemma, instead leading there with something like 
```lean
rw metric.nhds_basis_ball.tendsto_right_iff,
```
(this is where I was going with the "filter-y" comment, although I agree it's not bad as an explicit calculation).  I'll play with it tomorrow just to see.

Reply:
Thanks!

I feel like in the "filter-y" approach lemma discovery is nigh impossible (for a newcomer to this part of the library). At least back in the primitive world of inequality wrangling I know how to work out the names of lemmas... :-(